package.json

```diff
@@ -1,33 +1,52 @@
 {
   "name": "rds-contracts",
   "version": "0.0.1",
-  "description": "contracts for incentivized pinning",
-  "main": "truffle-config.js",
-  "directories": {
-    "test": "test"
+  "description": "Contracts for incentivized pinning",
+  "keywords": [
+    "RIF",
+    "RSK",
+    "pinning"
+  ],
+  "homepage": "https://github.com/rsksmart/rif-marketplace-storage-pinning",
+  "bugs": {
+    "url": "https://github.com/rsksmart/rif-marketplace-storage-pinning/issues"
   },
-  "scripts": {
-    "test": "truffle test"
+  "license": "MIT",
+  "author": {
+    "name": "Rinke Hendriksen",
+    "email": "<rinke@iovlabs.org>"
+  },
+  "files": [
+    "types"
+  ],
+  "directories": {
+    "contracts": "contracts",
+    "test": "test/"
   },
+  "main": "truffle-config.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/rsksmart/rds-contracts.git"
+    "url": "https://github.com/rsksmart/rif-marketplace-storage-pinning"
   },
-  "author": "Rinke Hendriksen <rinke@iovlabs.org>",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/rsksmart/rds-contracts/issues"
+  "scripts": {
+    "test": "truffle test",
+    "lint": "tasegir lint"
+  },
+  "dependencies": {
+    "dotenv": "^8.2.0",
+    "truffle-hdwallet-provider": "^1.0.17",
```

Comment:
These should be devDependency no?

Reply:
Yes and I actually removed some of the dependencies as we don't use them